pkg/sqlmigrations/migrations.go

```diff
@@ -780,3 +786,40 @@ func createDefaultDbs(ctx context.Context, r runner) error {
 	}
 	return err
 }
+
+func addJobsProgress(ctx context.Context, r runner) error {
```

Comment:
is there a test we can add somewhere to ensure that this migration has run? like a test that expects a specific output of `show create table system.jobs`?